File: codeforces/451A/go/451a.go
package main

import "fmt"

func play(n, m int) int {
    if m == 1 || n == 1 {
        return 1
    }
    return play(n-1, m-1) ^ 1
}

func main() {
    var n, m int
    fmt.Scan(&n, &m)

    if (play(n, m) & 0x1) == 0 {
        fmt.Println("Malvika")
    } else {
        fmt.Println("Akshat")
    }
}

File: codeforces/451A/py/451a.py
import sys

def play(n, m):
    if m == 0 or n == 0:
        return 1
    return play(n-1, m-1) ^ 1

n, m = map(int, sys.stdin.readline().split())
if play(n, m) % 2 == 0:
    print("Akshat")
else:
    print("Malvika")
